src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -68,4 +68,15 @@ private static function formatInterval(\DateInterval $i)
 
         return $i->format(rtrim($format));
     }
+
+    public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stub, $isNested, $filter)
+    {
+        $location = $timeZone->getLocation();
+        $formatted = (new \Datetime('now', $timeZone))->format($location ? 'e (P)' : 'P');
+        $title = $location && extension_loaded('intl') ? \Locale::getDisplayRegion('-'.$location['country_code'], \Locale::getDefault()) : '';
```

Comment:
just realized, does it make sense to use `Locale::getDefault()` instead of just `en`. Not sure this output needs to be localized at all :confused: